.pre-commit-config.yaml

```
@@ -9,8 +9,7 @@ repos:
       hooks:
           - id: black
             language_version: python3
-    - repo: https://github.com/pycqa/flake8
-      rev: 5.0.4
+    - repo: https://github.com/charliermarsh/ruff-pre-commit
```

```suggestion
    - repo: https://github.com/astral-sh/ruff-pre-commit
```
Looks like this changed recently (there's a redirect currently, but may as well move to the new url).